lib/market.rb

```diff
@@ -1,3 +1,80 @@
 class Market
- 
+require 'date'
+    
+    attr_reader :name,
+                :date
+
+    attr_accessor :vendors
```

Comment:
This could probably just be a attr_reader, as this vendor data doesnt need to be updated outside of this class